collections/choosing-projects/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 items:
  - rust-lang/rust
+ - golang/go
  - HospitalRun/hospitalrun-frontend
  - hoodiehq/hoodie
```

Comment:
[docs]    def account_media(self, id=None, **kwargs):
        """
        Returns a collection of account media available to the current account.
        """
        return self._load_resource(AccountMedia, id, **kwargs)